crates/core/tedge_api/src/device_id.rs

```diff
@@ -0,0 +1,36 @@
+pub fn new_get_child_id_from_topic(parent_device_name: String, topic: String) -> Option<String> {
+    if topic.starts_with("te/device") {
```

Comment:
We should make the root prefix (e.g. `te`) configurable. Though since we have multiple PRs going on at the moment, we can just put a `TODO` command above the prefix so that we know we need to change it (or alternatively, create a ticket for the follow up work).

Reply:
Though at the minimum we should at least create a constant for the root prefix until we are at a stage that we can read it from the tedge configuration.

Reply:
done